conf.py

```diff
@@ -307,7 +307,7 @@ def get_tool_name(envvar, default):
     if re.search(r'[^-/_+.A-Za-z0-9]', tool):
         # Or we could actually escape it...
         print("Bad characters in $%s" % envvar)
-        sys.exit(1)
+#        sys.exit(1)
```

Comment:
`-` is in there - the leading `^` means "not one of..." and the `-` after than is literal, not indicating a range.

If you were passing options, the "bad character" was probably a space - the current check assumes there aren't options included with the tool (which is probably flawed).